### DISCUSSION TITLE
ESPixelStick not working with 4.0-ci6339460065

### DISCUSSION THREAD

**Discussion by JMHaro** [forkineye/ESPixelStick#671] [General]
I started noticing issues with my ESPixelStick running from an FPP V7.1.  Sometimes it was working fine but other times I would find that it would just freeze and keep the lights at what ever they were doing at that time.  I've even seen that it didn't end the sequence when the FPP ended and just kept the lights on for the rest of the night.  It ran without any problems all spring and summer and I only updated the FPP's getting ready for the upcoming Halloween and Christmas seasons.  And when I say it freezes, the lights would stay fixed not following the sequence and the FPP was reporting that it was no longer seeing the ESPixelStick but the web pages were accessible from my main computer and within xLights it was see it also and could open the web page.

I was told that you might have a newer version that might fix issues with the FPP.  I had a hard time finding where to pull down the beta versions.  I did find the 4.0-ci6339460065 and I just tried to flash it, but afterwards the blue light never came back on (if it was supposed to).  The web site to it took a long time to respond but when it did, it displayed as text down the side of the page and everything in what it was showing like network settings area was all blank (attaching images if they help).

I had to go back to the 3.2 version (the blue light came back on with that version and web pages are back as normal).  

But, I would appreciate knowing if there is a possible known issue with the FPP V7.1 and if there is any fix for the issue and how/where can I get the correct update that might work.  I don't what the lights it's running to not be in sync with the FPP sequence being played.
![2023-10-02_17-00-30](https://github.com/forkineye/ESPixelStick/assets/3945942/796c4ef1-cc94-406f-894c-8e7ec0c938a3)
![2023-10-02_16-50-47](https://github.com/forkineye/ESPixelStick/assets/3945942/06ecc323-bb90-4499-8bb8-845edb0f5fbc)

**Comment by MartinMueller2003** [comment]
I recently upgraded my FPP5 devices to FPP 7 and I am seeing a ton of issues. For my show I will be going back to FPP 5 until I can work out the problems. For starters, the FPP has significantly reduced the number of sync messages. This may be fine in a wired environment, but in a WiFi setup, broadcast messages tend to get dropped and having more of them allowed the ESP to keep up. Now if a single stop or start message gets lost, then there is a flicker caused by the resync, sequence does not start etc. More as I figure things out.

**Reply by JMHaro** [reply]
Thanks Martin,
I will probably do the same thing then.  I think I was on 6.1.1 prior to upgrading which I had upgraded to that prior to last Christmas and never noticed the problem with that one, so might just go back to that one and try it again.  I do have one FPP that can only run Wifi due to it's location so wondering if that would would have had similar problems being synced from the player.
